src/main/java/edu/cornell/kfs/fp/businessobject/PurchasingDataBase.java

```diff
@@ -0,0 +1,525 @@
+     */
+    public void setShipFromPostalZipCode(String shipFromPostalZipCode) {
+      this.shipFromPostalZipCode = shipFromPostalZipCode;
+    }
+
+
+    /**
+     * @return the destinationCountryCode
+     */
+    public String getDestinationCountryCode() {
+      return destinationCountryCode;
+    }
+
+
+    /**
+     * @param destinationCountryCode the destinationCountryCode to set
+     */
+    public void setDestinationCountryCode(String destinationCountryCode) {
+      this.destinationCountryCode = destinationCountryCode;
+    }
+
+
+    /**
+     * @return the uniqueVATInvoice
+     */
+    public String getUniqueVATInvoice() {
+      return uniqueVATInvoice;
+    }
+
+
+    /**
+     * @param uniqueVATInvoice the uniqueVATInvoice to set
+     */
+    public void setUniqueVATInvoice(String uniqueVATInvoice) {
+      this.uniqueVATInvoice = uniqueVATInvoice;
+    }
+
+
+    /**
+     * @return the orderDate
+     */
+    public Date getOrderDate() {
+      return orderDate;
+    }
+
+
+    /**
+     * @param orderDate the orderDate to set
+     */
+    public void setOrderDate(Date orderDate) {
+      this.orderDate = orderDate;
+    }
+    
+
+    public void setOrderDate(String orderDate) {
+      if (StringUtils.isNotBlank(orderDate)) {
+        SqlDateConverter converter = new SqlDateConverter();
+        converter.setPattern("yyyyMMdd");
+        this.orderDate = (Date) converter.convert(Date.class, orderDate);
+      }
+    }
+
+
+    /**
+     * @return the itemDescriptor
+     */
+    public String getItemDescriptor() {
+      return itemDescriptor;
+    }
+
+
+    /**
+     * @param itemDescriptor the itemDescriptor to set
+     */
+    public void setItemDescriptor(String itemDescriptor) {
+      this.itemDescriptor = itemDescriptor;
+    }
+
+
+    /**
+     * @return the quantity
+     */
+    public Integer getQuantity() {
+      return quantity;
+    }
+
+    /**
+     * @param quantity the quantity to set
+     */
+    public void setQuantity(Integer quantity) {
+      this.quantity = quantity;
+    }
+
+    /**
+     * @param quantity the quantity to set
+     */
+    public void setQuantity(String quantity) {
+      this.quantity = Integer.parseInt(quantity);
+    }
+
+
+    /**
+     * @return the unitOfMeasure
+     */
+    public String getUnitOfMeasure() {
+      return unitOfMeasure;
+    }
+
+
+    /**
+     * @param unitOfMeasure the unitOfMeasure to set
+     */
+    public void setUnitOfMeasure(String unitOfMeasure) {
+      this.unitOfMeasure = unitOfMeasure;
+    }
+
+
+    /**
+     * @return the unitCost
+     */
+    public KualiDecimal getUnitCost() {
+      return unitCost;
+    }
+
+
+    /**
+     * @param unitCost the unitCost to set
+     */
+    public void setUnitCost(KualiDecimal unitCost) {
+      this.unitCost = unitCost;
+    }
+
+
+    /**
+     * @param unitCost the unitCost to set
+     */
+    public void setUnitCost(String unitCost) {
+      this.unitCost = new KualiDecimal(unitCost);
```

Comment:
Another setter that does not follow the pattern of some of the others for explicitly handling blank values.